### PR TITLE
Update analysis page

### PR DESCRIPTION
**Pull request by MarkMc1089**
Hi @ryanleggett and @thomas-owen - I've updated the analysis page with new content. Before I merge, any feedback?

### CODE REVIEW

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 1]
```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
...
-Starting from the source tables, we want to create the agreed outputs. Depending on the work, there are many approaches and tools we can use to do this.
+## Selection of methods
+
+The analysis first requires an appropriate method that can provide the desired outputs.
+
+*	Create a short-list of methods that could be applicable to the problem.
+*	Consider the pros and cons of each candidate method:
+  *	are there examples of this method being applied to the problem archetype?
+  *	will the available data support the method? If not, is it feasible to transform the data into a more suitable form?
+  *	will the tools available support the method?
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:18]: These are not indented when compiled locally

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 2]
```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
...
+*	Decide which methods are best suited to the problem:
+  *	often the best method will be clear
+  *	sometimes you may need to try a small number of methods before deciding on the overall best one
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:21]: These are not indented when compiled locally

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 3]
```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
...
+
+Keep notes of your decision-making and method trials for the final write-up of the initiative.
+
+## Application of methods
+
+Apply the selected model(s) with **RIGOUR**; it should be **R**epeatable, **I**ndependent, **G**rounded in reality, **O**bjective, **U**ncertainty-managed, and **R**obust.
+
+* **Repeatable**: For an analytical process to be considered ‘valid’ it might reasonably be expected that for the “same” inputs and constraints the analysis produces the “same” outputs. It is important to note that different analysts will consider the analytical problem differently, potentially resulting in differing results, however if any one approach is repeated the results should be as expected.
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:29]: different type of quotations here. Should that be or should they be consistent?

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 4]
```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
...
+* **Grounded in reality**: Quality analysis takes the commissioner and analyst on a journey as views and perceptions are challenged and connections are made between the analysis and its real consequences. Connecting with reality in this way guards against failing to properly grasp the context of the problem which is being analysed.
+* **Objective**: Effective engagement and suitable challenge reduces potential bias and enables the commissioner and the analyst to be clear about the interpretation of the analytical results.
+*	**Uncertainty-managed**: Uncertainties have been identified, managed, and communicated throughout the analytical process.
+*	**Robust**: Provide the analytical result in the context of residual uncertainty and limitations to ensure it is used appropriately.
+
+## Quality Assurance (QA)
+
+Check your analysis thoroughly and often, not as a last step before delivery.
+
+*	Documentation is important because:
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:40]: Not sure the first one needs a bullet point

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 5]
```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
...
+  *	it allows us to transfer knowledge about the analysis
+  *	it keeps the analysis transparent
+  *	it provides instructions on how to use any interactive output
+*	Good analysis structure helps:
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:44]: Again not sure it needs a bullet point. Not seen this done anywhere else in the document

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 6]
```diff
@@ -7,8 +7,64 @@ image-alt: Photo by Shubham Dhage on Unsplash
...
+  *	in avoiding errors
+  *	make it easier to check
+  *	maintain ongoing workflows
+  *	to re-use or re-purpose later
+*	Verify the outputs by:
+  *	testing your analysis, preferably in a repeatable and automated way
+  *	asking for peer review of all aspects, including code, text and decisions made
+* Validate your analysis by:
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:52]: Bullet point needed?

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd [hunk 7]
```diff
@@ -17,39 +73,40 @@ Starting from the source tables, we want to create the agreed outputs. Depending
...
 * Where possible, check results against other sources.
 * Other sources could be data from previous initiatives, Management Information dashboards or ePACT2.
 
 Additionally, you must be careful about visualising sensitive results. Please contact your critical friend or the Data Science Lead if you are unsure. Results for external audience or general public must be subjected to Statistical Disclosure Control [LINK] before release.
-Before pushing the code to GitHub, it's essential to ensure that the output is cleared, and that no sensitive information remains within it.
 
-### RAP considerations (LINK?)
+Before pushing any code to GitHub, it's essential to ensure that the output is cleared, and that no sensitive information remains within it.
+
+## RAP considerations (LINK?)
```
Comment (thomas-owen) [inline guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:102]: what are we doing about a RAP link?

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd (hunk 3 above)
Comment (MarkMc1089) [reply guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:29]: Removed quotes from "same" x2, seemed unnecessary.

File: guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd (hunk 7 above)
Comment (MarkMc1089) [reply guidance/03-analysis/analysis-steps/02-analysis-modelling.qmd:102]: Added useful RAP resources section with couple of links

**Pull request merged**